dlp/google/cloud/dlp_v2/gapic/enums.py

```diff
@@ -318,8 +318,9 @@ class CommonCharsToIgnore(enum.IntEnum):
           NUMERIC (int): 0-9
           ALPHA_UPPER_CASE (int): A-Z
           ALPHA_LOWER_CASE (int): a-z
-          PUNCTUATION (int): US Punctuation, one of !\"#$%&'()*+,-./:;<=>?@[\]^_``{|}~
-          WHITESPACE (int): Whitespace character, one of [ \t\n\x0B\f\r]
+          PUNCTUATION (int): US Punctuation, one of !"#$%&'()\*+,-./:;<=>?@[]^\_\`{\|}~
+          WHITESPACE (int): Whitespace character, one of [
+          :raw-latex:`\t`:raw-latex:`\n`:raw-latex:`\x`0B:raw-latex:`\f`:raw-latex:`\r`]
```

Comment:
@lukesneeringer This can't be right, and it makes the module un-importable.  Looks like this is a symptom of https://github.com/googleapis/gapic-generator/issues/2191.